### PR TITLE
Adding ledger tendermint provider

### PR DESCRIPTION
**Pull request by jleni**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /target/
 /providers/signatory-*/target
```
Comment (tarcieri) [inline .gitignore]: Oops, this is obsolete... mind deleting it?

**Comment by jleni** [comment]
@tarcieri done. I removed the entry from .gitignore

File: signatory-ledger-tm/src/lib.rs [hunk 2]
```diff
@@ -197,6 +162,7 @@ mod tests {
                 Ok(_sig) => {}
                 Err(e) => {
                     println!("Err {:#?}", e);
+                    panic!()
                 }
```
Comment (Liamsi) [inline signatory-ledger-tm/src/lib.rs]: Could this be changed to:
`signer.sign(&some_message).unwrap();`?

File: signatory-ledger-tm/Cargo.toml [hunk 3]
```diff
@@ -16,7 +16,7 @@ circle-ci = { repository = "tendermint/signatory" }
 
 [dependencies]
 lazy_static = "1"
-ledger-cosmos = "0.2.0"
+ledger-tendermint = "0.3.1"
```
Comment (Liamsi) [inline signatory-ledger-tm/Cargo.toml]: Shouldn't we just count up here? (`0.3.0`)

**Pull request merged**